src/app/clusters/fan-control-server/fan-control-server.cpp

```diff
@@ -50,6 +50,15 @@ static_assert(kFanControlDelegateTableSize <= kEmberInvalidEndpointIndex, "FanCo
 
 Delegate * gDelegateTable[kFanControlDelegateTableSize] = { nullptr };
 
+typedef struct PercentCurrentQTracking {
+    bool initialized;
+    bool moveStarted;
+    DataModel::Nullable<chip::Percent> startPercent;
+    DataModel::Nullable<chip::Percent> endPercent;
+    auto lastUpdateTime;
```

Comment:
Might be worth explicitly initialising to zero in case we get a current update without a setting update